basyx.aasservice/basyx.aasservice-core/src/test/java/org/eclipse/digitaltwin/basyx/aasservice/AasServiceSuite.java

```diff
@@ -158,14 +183,13 @@ public void getPaginatedSubmodelReferencesPaginated() {
 
 	@Test
 	public void updateThumbnail() throws FileNotFoundException, IOException {
-		AssetAdministrationShell shell = DummyAssetAdministrationShellFactory.createWithDefaultThumbnail();
-		AasService aasService = getAasService(shell);
+		setAasThumbnail(
```

Comment:
Why use this method?
Instead you should set using aasServiceWithThumbnail.setThumbnail, but with the dummyImageIS_B, since _A is already there